app.geteduroam.geteduroam.yml

```diff
@@ -0,0 +1,110 @@
+id: app.geteduroam.geteduroam
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: geteduroam-gui
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --system-talk-name=org.freedesktop.NetworkManager
+
+build-options:
+  env:
+    - GOBIN=/app/bin
+    - GOROOT=/usr/lib/sdk/golang
+
+modules:
+  - name: geteduroam
+    buildsystem: simple
+    build-commands:
+      - install -Dm00644 cmd/geteduroam-gui/resources/share/applications/app.geteduroam.Linux.desktop $FLATPAK_DEST/share/applications/app.geteduroam.geteduroam.desktop
+      - install -Dm00644 cmd/geteduroam-gui/resources/share/icons/hicolor/48x48/apps/app.geteduroam.Linux.png $FLATPAK_DEST/share/icons/hicolor/48x48/apps/app.geteduroam.geteduroam.png
+      - install -Dm00644 cmd/geteduroam-gui/resources/share/icons/hicolor/128x128/apps/app.geteduroam.Linux.png $FLATPAK_DEST/share/icons/hicolor/128x128/apps/app.geteduroam.geteduroam.png
+      - install -Dm00644 cmd/geteduroam-gui/resources/share/icons/hicolor/256x256/apps/app.geteduroam.Linux.png $FLATPAK_DEST/share/icons/hicolor/256x256/apps/app.geteduroam.geteduroam.png
+      - install -Dm00644 cmd/geteduroam-gui/resources/share/icons/hicolor/512x512/apps/app.geteduroam.Linux.png $FLATPAK_DEST/share/icons/hicolor/512x512/apps/app.geteduroam.geteduroam.png
+      - install -Dm00644 metainfo/app.geteduroam.geteduroam.metainfo.xml $FLATPAK_DEST/share/metainfo/app.geteduroam.geteduroam.metainfo.xml
+      - $GOROOT/bin/go build -mod=vendor -trimpath -buildmode=pie -o geteduroam-gui ./cmd/geteduroam-gui
+      - $GOROOT/bin/go build -mod=vendor -trimpath -buildmode=pie -o geteduroam-cli ./cmd/geteduroam-cli
+      - $GOROOT/bin/go build -mod=vendor -trimpath -buildmode=pie -o geteduroam-notifcheck ./cmd/geteduroam-notifcheck
+      - install -Dm00755 geteduroam-gui $FLATPAK_DEST/bin/geteduroam-gui
+      - install -Dm00755 geteduroam-cli $FLATPAK_DEST/bin/geteduroam-cli
+      - install -Dm00755 geteduroam-notifcheck $FLATPAK_DEST/bin/geteduroam-notifcheck
```

Comment:
i'm not sure if it makes sense to add this to the flatpak. This is a small binary that periodically should be run, e.g. by systemd timers or a cron job so that a notification can be shown.

Reply:
It's probably pointless to include `geteduroam-notifcheck`, but I was thinking since I am already creating a flatpak and building the other binaries, I might aswell add it. Because why not?

Reply:
True! We can figure out later how we want to integrate it